imagepicker/src/main/java/com/esafirm/imagepicker/helper/ExcludedMediaSingleton.java

```diff
@@ -0,0 +1,50 @@
+package com.esafirm.imagepicker.helper;
+
+import com.esafirm.imagepicker.model.Image;
+
+import java.io.File;
+import java.util.ArrayList;
+
+/**
+ * Code written by Qandeel Abbassi on 9/20/2018 at 7:03 PM.
+ */
+public class ExcludedMediaSingleton {
```

Comment:
This could be generate singleton for transferring large data between others activities/fragment to ImagePicker. 
Can you add `ImagePickerConfig.selectedImages` to this as well? 

The name could be renamed to something more general as well, and please omit "Singleton" from it.